OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -421,9 +421,16 @@
     <Compile Include="Traits\World\PaletteFromCurrentTileset.cs" />
     <Compile Include="Traits\World\PaletteFromFile.cs" />
     <Compile Include="Traits\World\PaletteFromRGBA.cs" />
-    <Compile Include="Traits\World\PathFinder.cs" />
-    <Compile Include="Traits\World\PathfinderDebugOverlay.cs" />
-    <Compile Include="Traits\World\PathSearch.cs" />
+    <Compile Include="Traits\World\Pathfinder\CellInfoLayerManager.cs" />
```

Comment:
Most of these aren't traits, so shouldn't be grouped in with the traits.  I suggest creating a top-level Pathfinder directory and namespace for them instead.